asv_bench/benchmarks/groupby.py

```diff
@@ -361,16 +361,15 @@ def time_category_size(self):
 class FillNA:
     def setup(self):
         N = 1500
```

Comment:
`N = 1500` seems to be high as it runs for 18s, maybe `N` around ~500-1000 would show the diff too.

Reply:
agree the benchmarks should run much faster, 100 or even smaller, it will show a difference in per terms which is what is measured.

Reply:
Done